src/components/screens/HistoryScreen/mappers/createAccount.tsx

```diff
@@ -41,62 +41,93 @@ export const mapCreateAccountHistoryItem = ({
   themeColors,
   isCreateExternalAccount,
 }: CreateAccountHistoryItemData): HistoryItemData => {
-  const { account, starting_balance: startingBalance } = operation;
+  const { account, starting_balance: startingBalance, funder } = operation;
   const isRecipient = !isCreateExternalAccount;
-  const paymentDifference = isRecipient ? "+" : "-";
+
+  // SENDER's view: Treat as a regular XLM payment
+  if (!isRecipient) {
+    const senderActionIcon = (
+      <Icon.ArrowCircleUp size={16} color={themeColors.foreground.primary} />
+    );
+    const senderIcon = (
+      <AssetIcon
+        token={{
+          type: AssetTypeWithCustomToken.NATIVE,
+          code: NATIVE_TOKEN_CODE,
+        }}
+        size="lg"
+      />
+    );
+    const transactionDetails: TransactionDetails = {
+      operation,
+      transactionTitle: `${String(t("history.transactionHistory.sent"))} ${NATIVE_TOKEN_CODE}`,
+      transactionType: TransactionType.PAYMENT,
+      fee,
+      status: TransactionStatus.SUCCESS,
+      IconComponent: senderIcon,
+      ActionIconComponent: senderActionIcon,
+      externalUrl: `${stellarExpertUrl}/op/${operation.id}`,
+      paymentDetails: {
+        to: account,
+        from: funder,
+        amount: startingBalance,
+        assetCode: NATIVE_TOKEN_CODE,
+        assetType: "native",
+      },
+    };
+
+    return {
+      transactionDetails,
+      rowText: t("history.transactionHistory.sent"),
+      dateText: date,
+      amountText: `-${formatAssetAmount(startingBalance, NATIVE_TOKEN_CODE)}`,
+      actionText: t("history.transactionHistory.sent"),
+      ActionIconComponent: senderActionIcon,
+      IconComponent: senderIcon,
+      transactionStatus: TransactionStatus.SUCCESS,
+      isAddingFunds: false,
+    };
+  }
+
+  // RECIPIENT's view: Show as "Account Funded"
+  const paymentDifference = "+";
   const formattedAmount = `${paymentDifference}${formatAssetAmount(
     startingBalance,
     NATIVE_TOKEN_CODE,
   )}`;
```

Comment:
I think we could remove this `paymentDifference` const and just add "+" directly on the `formattedAmount` string below since that's the only place using `paymentDifference` now. What do you think?